homeassistant/components/tuya/number.py

```diff
@@ -84,6 +84,36 @@
             icon="mdi:microphone",
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        NumberEntityDescription(
+            key=DPCode.PIR_DELAY,
+            name="PIR Delay",
+            native_unit_of_measurement="Secs",
```

Comment:
`sec` is the official abbreviation for seconds, however, we just use the full name: `seconds`.


```suggestion
            native_unit_of_measurement="seconds",
```